automod/rules/reposts.go

```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"time"
+
+	"github.com/bluesky-social/indigo/automod"
+	"github.com/bluesky-social/indigo/automod/countstore"
+)
+
+var repostDailyThreshold = 500
+
+var _ automod.RecordRuleFunc = TooManyRepostRule
+
+// looks for accounts which do frequent reposts
+func TooManyRepostRule(c *automod.RecordContext) error {
+
+	did := c.Account.Identity.DID.String()
+	// Don't bother checking reposts from accounts older than 30 days
+	if c.Account.Private != nil {
+		age := time.Since(c.Account.Private.IndexedAt)
+		if age > 30*24*time.Hour {
```

Comment:
I have this as 2 weeks in a couple other places, but 30 days is totally reasonable for this rule.

Reply:
cool yeah I was going with what Aaron recommended.